README.md

```diff
@@ -64,3 +68,32 @@ For more information, see the [Rust API embedding chapter] of the Wasmtime docum
 [Rust API embedding chapter]: https://bytecodealliance.github.io/wasmtime/embed-rust.html
 
 It's Wasmtime.
+
+#### Standalone runtime
+
+The [Releases](https://github.com/bytecodealliance/wasmtime/releases) page has pre-built binaries for each release, with binaries for Linux, macOS and Windows.
+
+If you want to build the latest Wasmtime from scratch, you will need:
+
+* An updated [Rust toolchain](https://www.rust-lang.org/install.html) installed
+* A C toolchain installed - e.g. the [`build-essential`](https://packages.ubuntu.com/xenial/build-essential) package in Ubuntu
+* A clone of the Wasmtime Git repo
+
+Example:
+
+```sh
+git clone --recursive https://github.com/bytecodealliance/wasmtime.git
+cargo build --release
+```
+
+This will create a binary at `./target/release/wasmtime`, which you can then invoke directly to run a given WASI-compatible WASM binary:
+
+```sh
+$ ./target/release/wasmtime YOUR-BINARY.wasm
```

Comment:
While this works, would it be better to use `cargo run --release` here, instead of `./target/release/wasmtime`?